public/index.html

```diff
@@ -6,6 +6,46 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Homepage</title>
 		<link rel="stylesheet" href="../src/css/index.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
 	</head>
-	<body></body>
+	<body>
+		<header>
+			<form>
+				<div class="list-navigation">
+					<div class="list-navigation__search-wrapper">
+						<input
+							type="search"
+							class="list-navigation__search-input"
+							placeholder="Search..."
+						/>
+						<button class="list-navigation__search-button" type="submit">
+							<i class="fa fa-search search-icon fa-lg"></i>
+						</button>
+					</div>
+					<div class="list-navigation__nav-links">
+						<ol class="list-navigation__nav-links__list">
+							<li class="list-navigation__nav-links__list--style">
+								<a
+									class="list-navigation__nav-links--link"
```

Comment:
when you use modifier you type class like this:
`<a class="list-navigation__nav-links list-navigation__nav-links--link"`
To have old style from class list-navigation__nav-links and new styles from list-navigation__nav-links--link